integration-tests/cucumber-tests-platform-common/src/test/resources/features/common/osgp-adapter-ws-admin/DeviceManagement/FilterDeviceLogItem.feature

```diff
@@ -0,0 +1,96 @@
+      | Organization         | TEST                 |
+
+  Scenario: No filters set so getting all the messages
+    When receiving a message log request without a filter
+    Then the messages response contains 15 messages with date filter or no filter
+
+  Scenario: Use wildcard filter for Device Identification
+    When receiving a filter message log request
+      | DeviceIdentification | DEV-1* |
+    Then the messages response contains 10 messages for devices
+      | DEV-1  |
+      | DEV-11 |
+
+  @NightlyBuildOnly
+  Scenario: Use wildcard filter for Organisation Identification
+    When receiving a filter message log request
+      | OrganizationIdentification | Li?nder |
+    Then the messages response contains 11 messages for devices
+      | DEV-1 |
+      | DEV-2 |
+
+  @NightlyBuildOnly
+  Scenario: Filter messages only on Organisation Identification
+    When receiving a filter message log request
+      | OrganizationIdentification | Liander |
+    Then the messages response contains 11 messages
+      | OrganizationIdentification | Liander |
+
+  Scenario: Filter messages on Organisation Identification and Device Identification
+    When receiving a filter message log request
+      | DeviceIdentification       | DEV-2   |
+      | OrganizationIdentification | Liander |
+    Then the messages response contains 5 messages
+      | DeviceIdentification       | DEV-2   |
+      | OrganizationIdentification | Liander |
+
+  Scenario: Sort messages by Device Identification
+    When receiving a filter message log request
+      | SortDir  | DESC                 |
+      | SortedBy | deviceIdentification |
+    Then the messages response contains 15 messages ordered descending by device identification
+
+  Scenario Outline: Filter messages on Organisation Identification and Device Identification and within two dates
+    When receiving a filter message log request
+      | OrganizationIdentification | Liander     |
+      | DeviceIdentification       | DEV-2       |
+      | StartTime                  | <StartTime> |
+      | EndTime                    | <EndTime>   |
+    Then the messages response contains <Amount> messages
+      | OrganizationIdentification | Liander |
+      | DeviceIdentification       | DEV-2   |
+
+    Examples: 
+      | StartTime            | EndTime              | Amount |
+      | 2020-01-01T00:00:00Z | 2020-05-01T00:00:00Z |      0 |
+      | 2020-01-01T00:00:00Z | 2040-01-01T00:00:00Z |      5 |
+
+  @NightlyBuildOnly
+  Scenario Outline: Filter messages within two dates
+    When receiving a filter message log request
+      | StartTime | <StartTime> |
+      | EndTime   | <EndTime>   |
+    Then the messages response contains <Amount> messages with date filter or no filter
+
+    Examples: 
+      | StartTime            | EndTime              | Amount |
+      | 2020-01-01T00:00:00Z | 2020-05-01T00:00:00Z |      0 |
+      | 2020-01-01T00:00:00Z | 2025-01-01T00:00:00Z |     15 |
```

Comment:
This will fail after 2025-01-01, I would suggest to use a different date like in the scenario above.

Reply:
Done (2025 => 2040).